lib/bbbevents/recording.rb

```diff
@@ -146,13 +148,44 @@ def timestamp_conversion(base)
 
     # Calculates an attendee's duration.
     def total_duration(att)
-      return 0 unless att.joins.length == att.leaves.length
-      total = 0
+      joins_leaves_arr = []
+      att.joins.each { |j| joins_leaves_arr.append({:time => j.to_i, :datetime => j, :event => :join})}
+      att.leaves.each { |j| joins_leaves_arr.append({:time => j.to_i, :datetime => j, :event => :left})}
+
+      #puts "BEGIN ============"
```

Comment:
Please remove the commented out/dead code.

Reply:
Removed.